packages/desktop-client/src/components/reports/spreadsheets/filterHiddenItems.tsx

```diff
@@ -0,0 +1,13 @@
+function filterHiddenItems(item, data) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏two things here:

1. missing argument types
2. the filename should be `.ts` instead of `.tsx` (`.tsx` implies using JSX; i.e. - a component. Whereas this is just a util function)